doc/ui/cmd_argument/detail_introduction.md

```diff
@@ -121,8 +121,12 @@
   - File that saves the model list when testing. It was set automatically when using cluster submitting environment after setting model_path.
   - type: string (default: "", null).
 
-* `--test_all_data_in_one_period`
-  - This argument is usually used in testing period during traning. If true, all data will be tested in one test period. Otherwise (batch_size * log_peroid) data will be tested.
+* `--test_batches_while_training`
+  - Test test_batches_while_training batches if test_batches_while_training != 0. If 0, test on all test data.
+  - type: bool (default: 1000).
```

Comment:
Test on test_batches_while_training batches data if not 0, else, test on all test data.